sdk/main/include/ExchangeRateSet.h

```diff
@@ -40,7 +40,7 @@ class ExchangeRateSet
   /**
```

Comment:
I'm not quite clear looking at this class what `ExchangeRateSet` is intended to represent. Would it be possible to make it a little more clear in the class doc?

Reply:
Done